tests/migrations/test_commands.py

```diff
@@ -1732,6 +1732,44 @@ def test_squashmigrations_squashes(self):
             '  you can delete them.\n' % squashed_migration_file
         )
 
+
+    def test_squashmigrations_squashes_already_squashed(self):
+        out = io.StringIO()
+
+        with self.temporary_migration_module(
+            module="migrations.test_migrations_squashed_complex"
+        ) as migrations_dir:
+            call_command(
+                "squashmigrations",
+                "migrations",
+                "3_squashed_5",
+                "--squashed-name",
+                "double_squash",
+                interactive=False,
```

Comment:
```suggestion
                interactive=False,
                stdout=out,
```

Because otherwise:
```
Found 1 test(s).
Creating test database for alias 'default'...
Creating test database for alias 'other'...
System check identified no issues (17 silenced).
Will squash the following migrations:
 - 1_auto
 - 2_auto
 - 3_squashed_5
Optimizing...
  No optimizations possible.
Created new squashed migration /var/folders/6q/jljmh5xs27v8_7557q3rgtfw0000gn/T/django_sgl9dr2z/tmpyeekddz7/tmpiuwbfzw6/migrations/0001_double_squash.py
  You should commit this migration but leave the old ones in place;
  the new migration will be used for new installs. Once you are sure
  all instances of the codebase have applied the migrations you squashed,
  you can delete them.
.
----------------------------------------------------------------------
Ran 1 test in 0.197s

OK
Destroying test database for alias 'default'...
Destroying test database for alias 'other'...
```